graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/spring/execution/ContextWebFilter.kt

```diff
@@ -37,6 +39,9 @@ const val GRAPHQL_CONTEXT_FILTER_ODER = 0
 open class ContextWebFilter<out T : GraphQLContext>(config: GraphQLConfigurationProperties, private val contextFactory: GraphQLContextFactory<T>) : WebFilter, Ordered {
     private val graphQLRoute = enforceAbsolutePath(config.endpoint)
     private val subscriptionsRoute = enforceAbsolutePath(config.subscriptions.endpoint)
+    private val parser = getPathPatternParser()
```

Comment:
should we move all of this calculation to `init` block so we only expose the actual patterns?

Reply:
Done